roles/deconz/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Check if the ZigBee device exists
```

Comment:
It would be good to follow this task up with a check and error if there are no devices found, rather than the `when` line at the bottom of the `docker_container` definition (which would fail silently).